test/emailService.spec.js

```diff
@@ -1,37 +1,39 @@
 const EmailService = require('../backend/services/emailService');
 const Email = require('../backend/models/email');
 const chai = require('chai');
-const chaiAsPromised = require('chai-as-promised');
-const assert = require('chai').assert;
+// const chaiAsPromised = require('chai-as-promised');
+const assert = chai.assert;
 
-chai.use(chaiAsPromised);
+// chai.use(chaiAsPromised);
 
 describe('Email Service Test Suite', () => {
   describe('Sending Emails...', () => {
     it('should send email and return promise with message ID', function() {
```

Comment:
No, it doesn't. It always times out - I messaged in the slack about this because a lot of articles like the [one that you linked before](https://medium.com/caffeine-and-testing/async-testing-with-mocha-with-callbacks-and-promises-5d0002661b3f) and another one [here](https://wietse.loves.engineering/testing-promises-with-mocha-90df8b7d2e35) suggest that calling done() isn't the best way to handle promises for tests. [Mocha's own website](https://mochajs.org/#working-with-promises) says that in version 3 and up, returning a promise and using done() throws an exception. How have you handled testing promises in the past? I saw that you used done() in the API tests but those were async calls and not promises, right?

Reply:
It doesn't matter if it's an aync call or a promise -- they're essentially the same. I'll pull your code down and try adding something and I'll push it if it worked.